ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -179,30 +176,12 @@ export class WorkspaceCard extends React.Component<WorkspaceCardProps, Workspace
     }
   }
 
-  // The function called when the 'share' action is called on a workspace card
-  // within the recentWorkspaces list.
-  async handleShareAction() {
```

Comment:
so this was all duplicated in WorkspaceShare already?

Reply:
yep, but only recently. I moved this login into the WorkspaceShare component and didn't realize there were other places where that action was available and had stuff to be cleaned up.